libkbfs/folder_branch_ops.go

```diff
@@ -523,7 +522,31 @@ func (fbo *folderBranchOps) setHeadLocked(ctx context.Context,
 			return nil
 		}
 
-		oldHandle = fbo.head.GetTlfHandle()
+		oldHandle := fbo.head.GetTlfHandle()
+		newHandle := md.GetTlfHandle()
+
+		resolvesTo, partialResolvedOldHandle, err :=
+			oldHandle.ResolvesTo(
```

Comment:
No, not with non-evil server/clients.

Do you think it's worth checking that the handles are equal first, so we can avoid `ResolvedTo` in the common case?  Or maybe the "TODO" in `ResolvedTo` about avoiding network access will make such an optimization useless?